wallet-new/integration/AccountSpecs.hs

```diff
@@ -98,6 +100,68 @@ accountSpecs wRef wc =
 
             map (AccountBalance . accAmount) accsUpdated `shouldBe` balancesPartialUpdated
 
+
+        prop "redeeming avvm key gives rise to the corresponding increase of balance of wallet'account - mnemonic not used" $ withMaxSuccess 1 $
+            monadicIO $ do
+
+            newWallet <- run $ randomWallet CreateWallet
+            Wallet{..} <- run $ createWalletCheck wc newWallet
+
+            --adding new account
+            rAcc <- pick arbitrary :: PropertyM IO NewAccount
+            newAcctResp <- run $ postAccount wc walId rAcc
+            newAcct <- run $ wrData <$> newAcctResp `shouldPrism` _Right
+
+            balancePartialRespB <- run $ getAccountBalance wc walId (accIndex newAcct)
+            balancesPartialB <- run $ wrData <$> balancePartialRespB `shouldPrism` _Right
+            let zeroBalance = AccountBalance $ V1 (Core.mkCoin 0)
+            liftIO $ balancesPartialB `shouldBe` zeroBalance
+
+            -- state-demo/genesis-keys/keys-fakeavvm/fake-9.seed
+            let avvmKey = "QBYOctbb6fJT/dBDLwg4je+SAvEzEhRxA7wpLdEFhnY="
+
+            --password is set to Nothing in the current implementation of randomWallet
+            --when it changes redemptionSpendingPassword handles it, otherwise passPhare addresses it
+            passPhrase <- pure mempty :: PropertyM IO SpendingPassword
```

Comment:
Minor note:

You can use `TypeApplication` or apply the type annotation to the `mempty` directly to avoid mentioning all the context around `pure (mempty @SpendingPassword)` or `pure (mempty :: SpendingPassword)`

Reply:
Actually, see note below.